### PR TITLE
chore: fix docs for datanode proto definitions

### LINKED ISSUE #7502: Add missing API documentation
Some of the endpoints are missing any description or information - meaning they're not very helpful for users.

### PR DESCRIPTION
**Pull request by AnExsomnis**
close #7502 
This PR edits/adds documentation across datanode API and is a WIP

### CODE REVIEW

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 1]
```diff
@@ -533,52 +533,57 @@ message PageInfo {
...
-    // formatted price of `1.23456` assuming market configured to 5 decimal places
-    // and importantly balances cannot be negative
+    // Asset balance represented as an integer, for example `123456` is a correctly
+    // formatted price of `1.23456` assuming market configured to 5 decimal places.
+    // Balances cannot be negative
     string balance = 3;
     // Asset identifier for the account
     string asset = 4;
-    // Market identifier for the account, if it's a general account, this will be empty
+    // Market identifier for the account, this field will be empty if the account is general
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:550]:
```suggestion
    // Market identifier for the account, this field will be empty the asset is in a general account
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 2]
```diff
@@ -787,24 +818,30 @@ message ObservePositionsResponse {
...
 // LedgerEntryFilter sets filters on returned set of ledger entries.
-// LedgerEntry objects can be filtered by asset_id, market_id, party_id for sending and receiving account, as well as on transfer types.
+// LedgerEntry data can be filtered by asset_id, market_id, party_id for sending and receiving account, as well as by transfer types.
 message LedgerEntryFilter {
     // close_on_account_filters determines whether an entry must have accounts matching both the account_from_filter
-    // and the account_to_filter. If true, entries must have matches in both filters.
+    // and the account_to_filter. If set to 'true', entries must have matches in both filters.
     // If false, entries matching only the account_from_filter or the account_to_filter will also be included.
     bool close_on_account_filters = 1;
+    // `AccountFilter` object used to set values for filtering in sender accounts
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:840]:
```suggestion
    // `AccountFilter` object used to set values for filtering sender accounts
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 3]
```diff
@@ -787,24 +818,30 @@ message ObservePositionsResponse {
...
     AccountFilter from_account_filter = 2;
+    // `AccountFilter` object used to set values for filtering in receiver accounts
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:842]:
```suggestion
    // `AccountFilter` object used to set values for filtering receiver accounts
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 4]
```diff
@@ -787,24 +818,30 @@ message ObservePositionsResponse {
...
     AccountFilter to_account_filter = 3;
+    // A list of `TrasnferType` that is used for filtering sender and receiver accounts
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:844]:
```suggestion
    // A list of `TransferType` that is used for filtering sender and receiver accounts
```

Comment (candida-d) [reply protos/sources/data-node/api/v2/trading_data.proto:844]: transfer typeS?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 5]
```diff
@@ -816,8 +853,10 @@ enum LedgerEntryField {
     LEDGER_ENTRY_FIELD_TRANSFER_TYPE = 3;
 }
 
+// Represents an aggregated view on ledger entries data, sender and receiver acounts,
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:856]:
```suggestion
// Represents an aggregated view of ledger entry data, sender and receiver accounts,
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 6]
```diff
@@ -847,39 +886,60 @@ message AggregatedLedgerEntry {
...
     LedgerEntryFilter filter = 1;
+    // Optional pagination control
     optional Pagination pagination = 2;
+    // Date range for which to list ledger entries
     optional DateRange date_range = 3;
 }
 
+// A request that is sent when exporting ledger entries to csv
 message ExportLedgerEntriesRequest {
+    // Party ID for which to export ledger entries
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:902]:
```suggestion
    // Party ID to export ledger entries for
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 7]
```diff
@@ -847,39 +886,60 @@ message AggregatedLedgerEntry {
...
     string party_id = 1;
+    // Asset ID for which to export ledger entries
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:904]:
```suggestion
    // Asset ID to export ledger entries for
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 8]
```diff
@@ -847,39 +886,60 @@ message AggregatedLedgerEntry {
...
     string asset_id = 2;
+    // Optional pagination control
     optional Pagination pagination = 3;
+    // Date range for which to export ledger entries
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:908]:
```suggestion
    // Date range to export ledger entries for
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 9]
```diff
@@ -847,39 +886,60 @@ message AggregatedLedgerEntry {
...
     bytes data = 1;
+    // Page information that is used for fetching further pages
     PageInfo page_info = 2;
 }
 
+// Represents the AggregatedLedgerEntry data with corresponding cursor for it
 message AggregatedLedgerEntriesEdge {
+    // 'AggregatedLedgerEntry' data
     AggregatedLedgerEntry node = 1;
+    // A cursor which can be used to fetch further pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:930]:
```suggestion
    // A cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 10]
```diff
@@ -889,15 +949,18 @@ message ListBalanceChangesRequest{
...
     AggregatedBalanceConnection balances = 1;
 }
 
 
 // -- Balances --
 
+// A request that is used when querying balance history
 message GetBalanceHistoryRequest {
-    // Limit the accounts considered according to the filter supplied
+    // AccountFilter contains values that can be set in order to limit balance history data for them
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:963]:
```suggestion
    // AccountFilter contains values that can be set in order to limit balance history data to them
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 11]
```diff
@@ -906,20 +969,29 @@ message GetBalanceHistoryRequest {
...
 message GetBalanceHistoryResponse {
+    // The list of aggregated balances with the list page info
     AggregatedBalanceConnection balances = 1;
 }
 
+// Represents the AggregatedBalanceEdge data with corresponding cursor for it
 message AggregatedBalanceEdge {
+    // AggregatedBalance data
     AggregatedBalance node = 1;
+     // A cursor which can be used to fetch further pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:982]:
```suggestion
     // A cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 12]
```diff
@@ -80,12 +80,12 @@ service TradingDataService {
...
     //
-    // Get ledger entries by asset, market, party, account type, transfer type within the given date range.
+    // List ledger entries by asset, market, party, account type and transfer type within the given date range.
     // This query requests and sums the number of ledger entries from a given subset of accounts, specified via the 'filter' argument.
     // It returns a time series (implemented as a list of AggregateLedgerEntry structs), with a row for every time
     // the summed ledger entries of the set of specified accounts changes.
-    // Listed queries should be limited to a single party from each side only. If no or more than one parties are provided
-    // for sending and receiving accounts - the query returns error.
+    // Listed entries should be limited to a single party from each side only. If zero or more than one parties are provided
+    // for each of the sides - sending and receiving accounts, the query returns error.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:88]:
```suggestion
    // for each of the sides - sending and receiving accounts, the query returns an error.
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 13]
```diff
@@ -533,52 +532,57 @@ message PageInfo {
...
 message AccountEdge {
-    // The actual account object
+    // The account balance data
     AccountBalance node = 1;
     // A cursor which can be used to fetch further pages with reference to this account
     string cursor = 2;
 }
 
-// Request to subscribe to a stream of accounts
+// Request that is sent when getting a subscribtion to a stream of accounts
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:585]:
```suggestion
// Request that is sent when getting a subscription to a stream of accounts
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 14]
```diff
@@ -847,39 +886,60 @@ message AggregatedLedgerEntry {
...
     string party_id = 1;
+    // Asset ID for which to export ledger entries
     string asset_id = 2;
+    // Optional pagination control
     optional Pagination pagination = 3;
+    // Date range for which to export ledger entries
     optional DateRange date_range = 4;
 }
 
+// A response that is received when lising ledger entries
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:912]:
```suggestion
// A response that is received when listing ledger entries
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 15]
```diff
@@ -906,20 +969,29 @@ message GetBalanceHistoryRequest {
...
 
+// Represents the list of AggregatedBalance objects with the list page info
 message AggregatedBalanceConnection {
+    // List of AggregatedBalance data chunks
     repeated AggregatedBalanceEdge edges = 1;
+    // Page information that is used for fetching further pages
     PageInfo page_info = 2;
 }
 
+// The filter that is used to filter account data by addet ID, party IDs, market IDs and account types
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:994]:
```suggestion
// The filter that is used to filter account data by asset ID, party ID, market ID and account type
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 16]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
 
+// Request that is sent for MarketDepth update subscription
 message ObserveMarketsDepthUpdatesRequest {
+    // List of market IDs to be obtained in the subscription
     repeated string market_ids = 1;
 }
 
+// Response that is received for Market Depth update subscription
 message ObserveMarketsDepthUpdatesResponse {
+    // Lis of market depth update data
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1055]:
```suggestion
    // List of market depth update data
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 17]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
+    // List of market data
     repeated vega.MarketData market_data = 1;
 }
 
+// Request that is sent when requesting latest market depth data
 message GetLatestMarketDepthRequest {
     // Market identifier, required field
     string market_id = 1;
 
+    // An optional filed to indicate the maximum market depth
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1076]:
```suggestion
    // An optional field to indicate the maximum market depth to request
```
is my addition correct?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 18]
```diff
@@ -1040,29 +1128,38 @@ message GetMarketDataHistoryByIDRequest {
...
 message GetMarketDataHistoryByIDResponse {
+    // The list of market data history with the list page info
     MarketDataConnection market_data = 1;
 }
 
+// Represents the MarketData data with corresponding cursor for it
 message MarketDataEdge {
+    // 'MarketData'
     vega.MarketData node = 1;
+    // Cursor that can be used to fetch futher pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1141]:
```suggestion
    // Cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 19]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
+// Response that is received when listing  transfers
 message ListTransfersResponse {
     TransferConnection transfers = 1;
 }
 
+// Represents the transfers data with a corresponding cursor for it
 message TransferEdge {
+    // Transfer data
     vega.events.v1.Transfer node = 1;
+    // Cursor that can be used to fetch futher pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1195]:
```suggestion
    // Cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 20]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
     CandleDataConnection candles = 1;
 }
 
+// Represents the candle data with the corresponding cursor for it
 message CandleEdge {
+    // List of candle data chunks
     Candle node = 1;
+    // Cursor that can be used to fetch futher pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1300]:
```suggestion
    // Cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 21]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
 message ListVotesResponse {
+    // Votes data received
     VoteConnection votes = 1;
 }
 
+// Votes data with the corresponding cursor for it
 message VoteEdge {
+    // Vote data
     vega.Vote node = 1;
+    // Cursor that can be used to fetch futher pages
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1335]:
```suggestion
    // Cursor that can be used to fetch further pages
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 22]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
 
 // -- Market Data --
 
-// Response for Market Data History
+// Request that is sent for getting MarketDepth subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1035]:
```suggestion
// Request that is sent to get market depth subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 23]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
     repeated string market_ids = 1;
 }
 
+// Response that is received for MarketDepth subscription
 message ObserveMarketsDepthResponse {
+    // List of market depth data
     repeated vega.MarketDepth market_depth = 1;
 }
 
+// Request that is sent for MarketDepth update subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1047]:
```suggestion
// Request that is sent for market depth update subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 24]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
 // -- Market Data --
 
-// Response for Market Data History
+// Request that is sent for getting MarketDepth subscription
 message ObserveMarketsDepthRequest {
+    // List of market IDs to be obtained in the subscription
     repeated string market_ids = 1;
 }
 
+// Response that is received for MarketDepth subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1041]:
```suggestion
// Response that is received for market depth subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 25]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
     repeated vega.MarketDepth market_depth = 1;
 }
 
+// Request that is sent for MarketDepth update subscription
 message ObserveMarketsDepthUpdatesRequest {
+    // List of market IDs to be obtained in the subscription
     repeated string market_ids = 1;
 }
 
+// Response that is received for Market Depth update subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1053]:
```suggestion
// Response that is received for market depth update subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 26]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
 message ObserveMarketsDepthUpdatesResponse {
+    // Lis of market depth update data
     repeated vega.MarketDepthUpdate update = 1;
 }
 
+// Request that is sent for MarketData subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1059]:
```suggestion
// Request that is sent for market data subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 27]
```diff
@@ -958,38 +1032,52 @@ message AggregatedBalance {
...
 message ObserveMarketsDataRequest {
+    // List of market IDs to be obtained in the subscription
     repeated string market_ids = 1;
 }
 
+// Response that is received for MarketData subscription
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1065]:
```suggestion
// Response that is received for market data subscription
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 28]
```diff
@@ -1040,29 +1128,38 @@ message GetMarketDataHistoryByIDRequest {
...
 }
 
-// Response for Market Data History
+// Response that is received when querying Market Data History
 message GetMarketDataHistoryByIDResponse {
+    // The list of market data history with the list page info
     MarketDataConnection market_data = 1;
 }
 
+// Represents the MarketData data with corresponding cursor for it
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1137]:
```suggestion
// Represents the market data with corresponding cursor for it
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 29]
```diff
@@ -1040,29 +1128,38 @@ message GetMarketDataHistoryByIDRequest {
...
 message MarketDataEdge {
+    // 'MarketData'
     vega.MarketData node = 1;
+    // Cursor that can be used to fetch futher pages
     string cursor = 2;
 }
 
+// Represents the list of MarketData objects with the list page info
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1145]:
```suggestion
// Represents the list of market data objects with the list page info
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 30]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
     TRANSFER_DIRECTION_TRANSFER_TO_OR_FROM = 3;
 }
 
-// Request for transfers
+// Request that is sent when listing transfers
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1176]:
```suggestion
// Request that is sent to list transfers
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 31]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
 message ListTransfersRequest {
-    // Public key of which transfers to request
+    // Public key of which to request transfers
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1178]:
```suggestion
    // Public key for which to request transfers
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 32]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
     optional string pubkey = 1;
-    // Directions to include in the response data
+    // Transfers directions to include in the response data
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1180]:
```suggestion
    // Transfer directions (sender, receiver) to include in the response data
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 33]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
 
+// Represents the transfers data with a corresponding cursor for it
 message TransferEdge {
+    // Transfer data
     vega.events.v1.Transfer node = 1;
+    // Cursor that can be used to fetch futher pages
     string cursor = 2;
 }
 
+// Represents the list of Transfers data objects with the list page info
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1199]:
```suggestion
// Represents the list of transfers data objects with the list page info
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 34]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
+    // List of transfers data chunks
     repeated TransferEdge edges = 1;
+    // Page information that is used for fetching further pages
     PageInfo page_info = 2;
 }
 
 // -- Network Limits --
 
-// Request for the current network limits
+// Request that is used when querying current network limits
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1209]:
```suggestion
// Request that is used to query current network limits
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 35]
```diff
@@ -1076,50 +1173,57 @@ enum TransferDirection {
...
 message GetNetworkLimitsRequest {}
 
-// Response for the current network limits
+// Response that is used when querying the current network limits
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1212]:
```suggestion
// Response received when querying the current network limits
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 36]
```diff
@@ -1128,8 +1232,9 @@ message IntervalToCandleId {
     string candle_id=2;
 }
 
-// A list of interval to candle id mappings for a given market
+// A list of mappings interval to candle ID for a given market
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1235]: Don't quite understand this one

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 37]
```diff
@@ -1128,8 +1232,9 @@ message IntervalToCandleId {
...
 message ListCandleIntervalsResponse {
+    // Mappings interval to candle ID
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1237]: I don't quite understand this one either, specifically "mappings interval"

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 38]
```diff
@@ -1152,17 +1257,19 @@ message Candle {
...
 }
 
+// Response that is received when subscribing to a stream of candles
 message ObserveCandleDataResponse {
+    // Candle data
     Candle candle = 1;
 }
 
-// Request for a list of candles for a market at an interval
+// Request that is used when listing candles for a market at an interval
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1272]:
```suggestion
// Request that is used to list candles for a market at an interval
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 39]
```diff
@@ -1152,17 +1257,19 @@ message Candle {
...
 message ListCandleDataRequest {
     // Candle identifier, required field.
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1274]: Candle ID maybe? for consistency

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 40]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
 
+// Represents the candle data with the corresponding cursor for it
 message CandleEdge {
+    // List of candle data chunks
     Candle node = 1;
+    // Cursor that can be used to fetch futher pages
     string cursor = 2;
 }
 
+// Represents the list of Candle data objects with the list page info
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1304]: What is "the list page info"?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 41]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
+    // List of candle data
     repeated CandleEdge edges = 1;
+    // Page information that is used for fetching further pages
     PageInfo page_info = 2;
 }
 
 // -- Votes --
 
-// Request for Votes
+// Request that is used for listing Votes
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1314]:
```suggestion
// Request that is used to list governance votes
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 42]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
 message ListVotesRequest {
-  // Party of which the votes are requested
+  // Party for which the votes are requested
   optional string party_id = 1;
-  // A proposal id for which top list proposal for
+  // A proposal ID for which top list proposal for
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1318]:
```suggestion
  // A proposal ID to list the votes for
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 43]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
 
+// Represents the list of Vote data chunks with the list page info
 message VoteConnection {
+    // List of Vote data
     repeated VoteEdge edges = 1;
+    // Page information that is used for fetching further pages
     PageInfo page_info = 2;
 }
 
+// Request that is sent when subscribing to votes
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1347]:
```suggestion
// Request that is sent to subscribe to votes
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 44]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
 message ObserveVotesRequest {
     // Party of which the votes are requested
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1349]:
```suggestion
    // Party ID for which the votes are requested
```

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 45]
```diff
@@ -1186,52 +1293,68 @@ message ListCandleDataResponse {
...
     optional string party_id = 1;
     // Option proposal id for which the votes are requested
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:1351]:
```suggestion
    // Option proposal ID for which the votes are requested
```

**Review by candida-d — COMMENTED** [review]
some typo fixes, bit of rewording and some questions

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 46]
```diff
@@ -601,50 +606,61 @@ message ObserveAccountsResponse {
 
 // An 'initial image' snapshot containing current account balances - may be sent over several response messages
 message AccountSnapshotPage{
+    // List of account balance objects
```
[the remark this answers is not in the document]
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:609]: data chunk - that is represented by 'AccountBalance' type

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 47]
```diff
@@ -601,50 +606,61 @@ message ObserveAccountsResponse {
...
+// A response that is received from the query for getting a single order
 message GetOrderResponse {
-    // An order, if found
+    // The order, if it was found
     vega.Order order = 1;
 }
 
+// Order filter that contains all filtering conditions and values that are applies to the orders listing
 message OrderFilter {
+    // The list of order statuses the orders should be filtered on
```
[the remark this answers is not in the document]
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:653]: Yes, I changed it to include "can"

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 48]
```diff
@@ -654,47 +670,52 @@ message ListOrdersRequest {
...
     optional OrderFilter filter = 7;
 }
 
+// A response that is received from the query of listing orders
 message ListOrdersResponse {
     // Paged connection with 0 or more orders
     OrderConnection orders = 1;
 }
 
+// A request that is sent when listing of order versions
```
[the remark this answers is not in the document]
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:687]: Yes.
Added "possible", is that helpful?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 49]
```diff
@@ -706,18 +727,21 @@ message ObserveOrdersResponse {
 
 // 'Initial image' of live orders, may be sent over multiple response messages
 message OrderSnapshotPage{
+    // List of 'Order' objects
```
[the remark this answers is not in the document]
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:730]: data chunks
changed it to "orders data parts", is it clear in this way?

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 50]
```diff
@@ -738,44 +762,51 @@ message ListPositionsResponse {
...
 
+// Represents the position data with corresponding cursor for it
 message PositionEdge {
+    // Represents position data for a party on the specified market on Vega
     vega.Position node = 1;
+    // A cursor which can be used to fetch further pages
     string cursor = 2;
 }
 
+// Represents the list of positions with the list page info
```
[the remark this answers is not in the document]
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:793]: This part of the struct: " PageInfo page_info = 2;"

File: protos/sources/data-node/api/v2/trading_data.proto (hunk 4 above)
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:844]: yes, that would be better, changed.

File: protos/sources/data-node/api/v2/trading_data.proto (hunk 36 above)
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:1235]: It means a list of mapped pairs "interval" -> "candle ID"

File: protos/sources/data-node/api/v2/trading_data.proto (hunk 37 above)
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:1237]: Same as the above comment explanation, but this is the filed that contains it

File: protos/sources/data-node/api/v2/trading_data.proto (hunk 40 above)
Comment (AnExsomnis) [reply protos/sources/data-node/api/v2/trading_data.proto:1304]: it means the page info content for the list - it tells you on which page you are in the list of candles

File: protos/sources/data-node/api/v2/trading_data.proto [hunk 51]
```diff
@@ -533,52 +532,57 @@ message PageInfo {
...
-// Represents an account for an asset on Vega for a particular owner or party
+// Represents the current balance of an account for an asset on Vega, for a particular owner or party
 message AccountBalance {
     // Unique account identifier (used internally by Vega)
     // string id = 1;
-    // The party that the account belongs to, special values include `network`, which represents the Vega network and is
-    // most commonly seen during liquidation of distressed trading positions
+    // The party that owns the account.
+    // Special values include `network` - represents the Vega network and is
+    // the most commonly seen during liquidation of a distressed trading position
```
Comment (candida-d) [inline protos/sources/data-node/api/v2/trading_data.proto:541]:
```suggestion
    // most commonly seen during liquidation of a distressed trading position
```

**Review by jeremyletang — APPROVED** [review]
Well, you've probably made an @candida-d very happy with that one!

**Pull request merged**